articles/governance/policy/how-to/remediate-resources.md

```diff
@@ -65,20 +65,62 @@ az role definition list --name 'Contributor'
 
 ## Manually configure the managed identity
 
-When creating an assignment using the portal, Azure Policy both generates the managed identity and
+When creating an assignment using the portal, Azure Policy can both generate a managed identity and
 grants it the roles defined in **roleDefinitionIds**. In the following conditions, steps to create
 the managed identity and assign it permissions must be done manually:
 
 - While using the SDK (such as Azure PowerShell)
 - When a resource outside the assignment scope is modified by the template
 - When a resource outside the assignment scope is read by the template
 
+## Configure a managed identity through Portal
+
+When creating an assignment using the portal, it is possible to select either a system or user assigned managed identity. 
+
+To set a system assigned managed identity through portal follow these steps: 
+
+1. In the **Remediation** tab of the create/edit assignment view, ensure that **System assigned managed identity** 
+is selected under "Types of Managed Identity". 
+
+1. Specify the location at which the managed identity is to be located. 
```

Comment:
The numbering for the steps looks incorrect. Same goes for the steps for assigning a "User assigned managed identity"

Reply:
It seems that the docs will auto change the numbers. I believe this a doc pattern to just name everything one.